src/Commands/Env/SyncCodeCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Env;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Commands\WorkflowProcessingTrait;
+use Pantheon\Terminus\Exceptions\TerminusException;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+
+/**
+ * Class SyncCodeCommand.
+ *
+ * @package Pantheon\Terminus\Commands\Env
+ */
+class SyncCodeCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+    use WorkflowProcessingTrait;
+
+    /**
+     * Sync code for the given environment (only dev and multidev allowed).
+     *
+     * @authorize
+     *
+     * @command env:sync-code
```

Comment:
`env:rebuild` maybe? (Naming things is hard.)

Reply:
I agree that "rebuild" could sound risky.. and on the other side it might sound like it does more than it actually does.  The dashboard does say `Synced code on "dev"`.  I also feel like "sync-code" isn't super clear--it's a bit "inside baseball"--but I don't have a better suggestion.. I think it's clearer than "rebuild"

Reply:
I think that documentation can solve these problems. `sync-code` now runs Composer install again, so I think that name is a little outdated, and rebuild is better. Probably a product decision.